.azuredevops/build-and-push-dev.yml

```diff
@@ -5,7 +5,10 @@ variables:
   azureContainerRegistry.repository: 'eligibilityestimator'
   azureContainerRegistry.name: 'DTSShared-December/2023'
   azureContainerRegistry.domain: 'dtsshared.azurecr.io'
-  tag: 'ADO-$(Build.BuildId)'
+  app.environment: development
+  app.buildDate: $[ format('{0:yyyy}-{0:MM}-{0:dd}', pipeline.startTime) ]
+  app.domain: 'https://app-eligibilityestimator-dev.azurewebsites.net/'
```

Comment:
https://ep-be-staging.bdm-dev-rhp.dts-stn.com/   or 
https://ep-be-dev.bdm-dev-rhp.dts-stn.com/    or 
https://ep-be-dyna-pipeline-secrets.bdm-dev-rhp.dts-stn.com/